gpkit/nomials/nomial_math.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, exps=None, cs=1, require_positive=True, simplify=True,
                         exp[VarKey(key)] = exp.pop(key)
             else:
                 raise TypeError("could not make Monomial with %s" % type(exps))
-            #simplify = False #TODO: this shouldn't require simplification
+            #simplify = False
```

Comment:
Derp! I think I just convinced myself it did require simplification, and so deleted the comment. This all gets replaced by nomialmap.